packages/address/src/utils.ts

```diff
@@ -23,7 +23,8 @@ export const FUEL_BECH32_HRP_PREFIX = 'fuel';
 
 /**
  * Decodes a Bech32 address string into Decoded
- *
+ * @deprecated
+ * The `bech32Address` is now deprecated. Please migrate to `toB256` as this will be the standard going forward.
```

Comment:
Maybe we can change this notice here just to say that this helper will cease to exist as it is not going to be required anymore when we drop bech32.